### PR TITLE
12.0 mig partner contact nationality

### PR DESCRIPTION
**Pull request by tglukhov**

### CODE REVIEW

**Comment by andreschenkels** [comment]
Can this PR be merged? I would also like to port this to 13

**Comment by HaraldPanten** [comment]
> Can this PR be merged? I would also like to port this to 13

Hi @aschenkels-ictstudio ,

It has to be validated first until it's aproved. Why don't you review that?

THX!

**Comment by andreschenkels** [comment]
Your right. I'll do this.

Verzonden vanuit Outlook Mobile<https://aka.ms/blhgte>

________________________________

File: partner_contact_nationality/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
```
Comment (NL66278) [inline partner_contact_nationality/__init__.py:1]: Please no copyright in init files and make license lines https:

**Review by NL66278 — APPROVED** [review]
:+1: LGTM. Would be nice if you could update some of the headers (no copyright in init files, https for all license lines. Copyright lines should not have (C)). But not blocking.

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**